exe-unit/src/service/transfer.rs

```diff
@@ -100,7 +100,7 @@ impl ContainerTransferProvider {
             }
             Ok(vol_base.join(path))
         } else {
-            log::warn!("not found!!");
+            log::warn!("path {} not found in container", container_path);
```

Comment:
```suggestion
            log::warn!("path not found in container: {}", container_path);
```
it makes it less obscure if path is really long.

Reply:
Makes sense, but in that case, IMHO it also makes sense to tweak the error message below.